INSTALL.md

```diff
@@ -2,12 +2,58 @@
 
 ## Linux for testing
 
-Install python 2.x (will probably work with 3.x with little modification)
+Install python 2.x (will probably work with 3.x with little modification).
 
-Install requirements:
+Install pip and other dependencies:
```

Comment:
If you `git rebase master` it should get rid of them if your master is up to date. Please also do a `git rebase -i master` and reword and squash some of your commits. You can then `git push --force-with-lease` onto this remote branch to update the PR. https://www.atlassian.com/git/tutorials/rewriting-history#git-rebase